zuul.d/validations.yaml

```diff
@@ -78,7 +78,9 @@
     - examples/dt/uni01alpha
     - examples/dt/uni01alpha/control-plane
     - examples/dt/uni01alpha/control-plane/nncp
+    - examples/dt/uni01alpha/edpm
```

Comment:
Should `examples/dt/uni01alpha/edpm/nodeset` be included here as well?

Reply:
I dont have `examples/dt/uni01alpha/edpm/nodeset`

Reply:
`create-zuul-jobs.py` creates and validates these files. I believe it is working correctly.

Reply:
Ack